forage-android/src/main/java/com/joinforage/forage/android/core/ui/element/ForagePinElement.kt

```diff
@@ -78,6 +81,13 @@ abstract class ForagePinElement @JvmOverloads constructor(
         vault.clearText()
     }
 
+    override fun showKeyboard() = vault.showKeyboard()
```

Comment:
Can we either adjust `showKeyboard` to be
```kotlin
override fun showKeyboard() {
    vault.showKeyboard()
}
```
or adjust `clearText` to be 
```kotlin
override fun clearText() = vault.clearText()
```

It seems inconsistent to have one be a standard function and the other be assigned with `=`. I don't have a preference for which style we use, just feels like it should be consistent between the two.

Reply:
oooooo yeah that's a good point. I'll add that in a follow on-commit before merging. Thanks for flagging

Reply:
Just updated to use `{...}`!